### ISSUE TITLE
Android.xml

### ISSUE THREAD

**Issue by devz3ro** [zynine-/HDHRViewerV2.bundle#28]
What I'm running:
HDHomeRun: HDTC-2US 20161012beta1
Plex Media Server: plexmediaserver-1.2.3.2914-1ff0f18 (plex pass)

I have a Android TV (Google ADT-1), and this plugin only works when I set Passthrough audio to auto, and when I create the Profiles directory in the PMS data root, and put the Android.xml file you provided there. It works, and it works very well :-).

Unfortunately, while the Android.xml allows playback for this plugin, it breaks the playback of all my other media :-/. Is there anyway to force mp3 transcoding **without** using the Android.xml? 

I've played around with the below XML line in the Androd.xml file. I made sure to restart the PMS process every time during my testing to ensure it was reloaded. Unfortunately my local media **only** plays correctly when audioCodec="aac". I changed the container to "mkv" and tried audioCodec: he-aac, ac3, eac3, mp2, mp3, vorbis, flac, alac, wma, and pcm. The only codecs that don't error out are flac and alac, but unfortunately they have no sound.

`<VideoProfile protocol="hls" container="mpegts" codec="h264" audioCodec="mp3" context="streaming">`

In short: My local media will only play (with sound) when audioCodec="aac", and the HDHRViewerV2 plugin will only play when audioCodec="anythingbutaac"

I'm looking for a happy medium.

**Comment by zynine-** [comment]
Hmm, looks like I have a similar problem. My PMS 1.2.2 refuzses to play any media with AAC audio, while my Dev PMS 1.1.4 works.

**Comment by zynine-** [comment]
Can you try this profile instead? 
[Android.zip](https://github.com/zynine-/HDHRViewerV2.bundle/files/554533/Android.zip)

**Comment by devz3ro** [comment]
Perfect! thanks so much, now everything is working. It looks like the original one works if you strip out the dash element, I didn't even think of that. I modified the file you just attached to this for deinterlace purposes. Thanks again!

```
    <VideoProfile protocol="hls" container="mpegts" codec="h264" audioCodec="mp3" context="streaming">
      <Setting name="VideoEncodeFlags" value="-deinterlace -x264opts cabac=0" />
    </VideoProfile>
```

**Comment by zynine-** [comment]
No prob. Plex changed the profile in 1.2.x, so i ended up using the latest profile and modifying it instead.
